server/controllers/recommendations/dateValidation/formValidator.test.ts

```diff
@@ -0,0 +1,25 @@
+import { validateDateTime } from './formValidator'
+
+describe('validateDateTime', () => {
+  const recommendationId = '34'
+  const urlInfo = {
+    currentPageId: 'appointment-no-recall',
+    basePath: `/recommendations/${recommendationId}/`,
+    path: `/recommendations/${recommendationId}/appointment-no-recall`,
+  }
+
+  it('returns valuesToSave and no errors, and redirects to preview, if valid and other tasks complete', async () => {
```

Comment:
Is there a test for what happens if it's not valid?

Reply:
I have added a couple of tests for invalid date time values